src/js/components/landing/recentActivity/DeleteModal.jsx

```diff
@@ -33,13 +34,15 @@ export default class DeleteModal extends React.Component {
 						this.props.closeModal();
 					});
 				}
-			})
-			.catch((error) => {
-				this.setState({
-					disable: true,
-					errorMessage: (error.httpStats === 400 || error.httpStatus === 403) ? error.message : "An error occurred while attempting to delete the submission. Please contact your administrator for assistance."
-				});
+			}).catch((error) => {
+				console.log('CATCH', error);
+				let errorMessage = "";
+					errorMessage = error.message
```

Comment:
why do this in two lines? just say `let errorMessage = error.message'
Will `error.message` ever be null or empty? Should have a default message if there is nothing there